openreview/tools.py

```diff
@@ -407,11 +408,12 @@ def subdomains(domain):
         full_domain = domain.split('@')[1]
     else:
         full_domain = domain
-    domain_components = full_domain.split('.')
+    domain_components = [c for c in full_domain.split('.') if c and not c.isspace()]
     domains = ['.'.join(domain_components[index:len(domain_components)]) for index, path in enumerate(domain_components)]
-    valid_domains = [d for d in domains if '.' in d]
+    valid_domains = [d for d in domains if not tld.is_tld(d)]
     return valid_domains
 
+@deprecated(version='0.9.24')
```

Comment:
is it possible to remove these deprecated functions?

Reply:
I don't see why not - it might break something but we'll find out!